src/main.js

```diff
@@ -36,6 +36,12 @@ Vue.prototype.$eventBus = mitt()
 Vue.config.productionTip = false
 
 /* eslint-disable no-new */
+/*
+This is where the main app is created:
+- What are all the objects (?) being fed in?
+- What is render (h) doing?
+- What is ``$mount('#app')``?
+*/
```

Comment:
@oliver-sanders  are these valid questions to leave in the docs?
```suggestion
```

Reply:
* The "router" is the bit that translates URLs into views (e.g. `workflows` goes to the Workflow view)
* The "store" is the VueX store instance.
* "Vuetify" is a component library (we use it for buttons etc).
* "render" is the Vue render function.
* "$mount('#app')" causes the vue instance to be "mounted" on the HTML element with the ID "app"

Reply:
Again, these were added by `yarn install`. Should I edit to actually answer the questions?

Reply:
Delete.